package/MDAnalysis/core/universe.py

```diff
@@ -1122,7 +1157,7 @@ def add_bonds(self, values, types=None, guessed=False, order=None):
         .. versionadded:: 1.0.0
         """
         self._add_topology_objects('bonds', values, types=types,
-                                 guessed=guessed, order=order)
+                                   guessed=guessed, order=order)
```

Comment:
```suggestion
                                 guessed=guessed, order=order)
```

Revert unrelated change.